packages/trackerless-network/test/unit/ExternalNetworkRpc.test.ts

```diff
@@ -23,7 +23,7 @@ describe('ExternalNetworkRpc', () => {
     })
 
     it('registers method', async () => {
-        rpc.registerRpcMethod(HandshakeRequest, HandshakeResponse, 'handshake', () => fn())
+        rpc.registerRpcMethod(StreamPartHandshakeRequest, StreamPartHandshakeResponse, 'handshake', () => fn())
```

Comment:
Is this change intentional? Maybe it is, if the point here is that we shouldn't register `dht's` methods in `trackerless-network` package?

Could mention this change in the PR description (or extract this change to a separate PR).

Reply:
The DhtRpc is no longer imported to the .proto file in trackerless. Therefore, the test needed new foobar messages to use

Reply:
Good to update to PR description. This big PR is just about moving `PeerDescriptor`, and it is not trivial to know all prerequisites/consequences related to that.